src/hadoop-data-node/config/hadoop-data-node.md

```diff
@@ -0,0 +1,44 @@
+## Hadoop data node section parser
+
+- [Default Configuration](#D_Config)
+- [How to Configure](#HT_Config)
+- [Generated Configuraiton](#G_Config)
+- [Data Table](#T_config)
+
+#### Default configuration <a name="D_Config"></a>
+
+[hadoop-data-node default configuration](hadoop-data-node.yaml)
+
+#### How to configure cluster section in service-configuraiton.yaml <a name="HT_Config"></a>
+
+All configurations in this section is optional. If you want to customized these value, you can configure it in service-configuration.yaml.
+   
+- `storage_path` The hdfs storage folders, support comma-delimited list of directories. 
+if isn't specified, will use `cluster.common.data-path/hdfs/data`
+
+
+
+#### Generated Configuration <a name="G_Config"></a>
+
+After parsing, object model looks like:
+```yaml
+storage_path: /path/to/folder1,/path/to/folder2,etc.
```

Comment:
,etc. could remove which maybe misleading user. if the format is the same with hdfs-site.xml, could refer that config intro format

Reply:
update some comments